### DISCUSSION TITLE
Entry type field browser - allow sorting by field type

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#15081] [Ideas]
Right now, fields list is just sorted alphabetically. However i sometimes want to quickly filter/sort the list by field type. For example when i remember that field of specific type exist, but i have no idea what it is called.

**Comment by brandonkelly** [comment]
Worth noting that the search input added in Craft 5 does look at the Type column data, so for example you could type “Matrix” and see all the Matrix fields.
